lua/lspconfig/util.lua

```diff
@@ -102,23 +102,24 @@ M.path = (function()
     return path:gsub('([%[%]%?%*])', '\\%1')
   end
 
-  --- @param filename string
-  --- @return string|false
-  local function exists(filename)
-    local stat = uv.fs_stat(filename)
-    return stat and stat.type or false
-  end
-
   --- @param filename string
   --- @return boolean
   local function is_dir(filename)
-    return exists(filename) == 'directory'
+    local stat = uv.fs_stat(filename)
+    if not stat then
+      return false
+    end
+    return stat.type == 'directory'
```

Comment:
If we are going to insist that `vim.fs.exists` / `vim.fs.isdir` / etc. isn't needed, let's demonstrate how to use `fs_stat` in a non-verbose way :)

```suggestion
    return stat and stat.type == 'directory'
```